src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -519,18 +525,18 @@ class ProjectReview extends Component {
   };
 
   handleApproveDialog = () => {
-      this.setState({
-        showApproveDialog: !this.state.showApproveDialog,
-        editedForm: {},
-        errorSubmit: false
-      });
+    this.setState({
+      showApproveDialog: !this.state.showApproveDialog,
+      editedForm: {},
+      errorSubmit: false
+    });
```

Comment:
Here is another case where a current state calculation depends on previous state so it should use the updater form.